contribute.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Commonware",
+    "description": "A collection of small but useful tools for Django.",
+    "repository": {
+        "url": "https://github.com/jsocol/commonware",
```

Comment:
Has moved to `https://github.com/mozilla/commonware`. Also 👍 for adding this file!! 🥇